src/server/src/routes/users.js

```diff
@@ -113,11 +114,81 @@ router.post("/activation", async (req, res) => {
       console.log("Activation success");
     } catch (err) {
       console.log(err);
-      return res.status(400).json({ error: "Incorrect or expired link" });
+      return res.send("Incorrect or expired link" );
     }
   } else {
     console.log("activation error");
-    return res.json({ error: "Activation error" });
+    return res.send("Activation error");
+  }
+});
+
+// Email if user forgot password
+router.post('/send-forgot-password-email', async (req, res) => {
```

Comment:
this handler function is huge. you have bits of code that are repeating in different parts of the file. - find user, encrypt pasword etc. would recommend to move them out to reusable functions

Reply:
moved out some of the code, what do you think?